partner_contact_lang/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Manage language in contacts',
+    'version': '11.0.1.0.0',
+    'category': 'Sales',
+    'license': 'AGPL-3',
+    'author': 'Tecnativa,'
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact/',
+    'depends': [
+        'base',
```

Comment:
@cristinamartinrod `base` is a default depends, It's Remove it.

Reply:
You always need to depend at least in this one. If not, bad things can happen.